assets/panel/debugger.properties

```diff
@@ -166,9 +136,9 @@ sourceSearch.search.key=F
 # the search for re-searching the same search triggered from a sourceSearch
 sourceSearch.search.again.key=G
 
-# LOCALIZATION NOTE (sourceSearch1.resultsSummary): Shows a summary of
+# LOCALIZATION NOTE (sourceSearch.resultsSummary): Shows a summary of
 # the number of matches for autocomplete
-sourceSearch1.resultsSummary=%d instances of “%S”
+sourceSearch.resultsSummary=%d instances of “%S”
```

Comment:
Are you sure about this? The current string in mozilla central is sourceSearch.resultsSummary:

https://hg.mozilla.org/mozilla-central/file/tip/devtools/client/locales/en-US/debugger.properties#l129

(that's one of the things we changed in Bug 1334154)